examples/ticks_and_spines/centered_ticklabels.py

```diff
@@ -34,7 +35,7 @@
 ax.plot(date, r.adj_close)
 
 ax.xaxis.set_major_locator(dates.MonthLocator())
-ax.xaxis.set_minor_locator(dates.MonthLocator(bymonthday=15))
+ax.xaxis.set_minor_locator(dates.MonthLocator(bymonthday=16))
```

Comment:
The center between day 1 of two consecutive months is
- 1 + 30/2 = 16 for 30-day months.
- 1 + 31/2 = 16.5 for 31-day months.

The labels are not exactly aligned for 31-day months (and February), but 16 is a far better estimate than 15. :smile: 

That said, the example is not particularly good, because the widths vary and to be strictly centered, we would have to make a more complicated minor locator. - But with this patch the effect is at least less obvious.

Reply:
Might as well comment about it for the future:
```suggestion
# 16 is a slight approximation since months differ in number of days.
ax.xaxis.set_minor_locator(dates.MonthLocator(bymonthday=16))
```